rudder-webapp/SOURCES/rudder-node-to-relay

```diff
@@ -57,201 +76,25 @@ fi
-
-${ECHO} "# $0: Transforming ${RELAY_UUID} (${RELAY_HOSTNAME}) into a Rudder relay server on `date`" > ${TMPFILE}
-${LDAPSEARCH} -s base -b "nodeId=${RELAY_UUID},ou=Nodes,cn=rudder-configuration" >> ${TMPFILE}
-${LDAPDELETE} "nodeId=${RELAY_UUID},ou=Nodes,cn=rudder-configuration" >/dev/null
-
-${ECHO} " Done."
-
-#######################################################################################################################
-## Switching the node to Policy server type 
-#######################################################################################################################
-
-${ECHO} -n "INFO: Changing the machine type to relay server..."
-${LDAPADD} <<EOF >/dev/null 2>&1 || echo -n " entries already exist."
-dn: nodeId=${RELAY_UUID},ou=Nodes,cn=rudder-configuration
-objectClass: top
-objectClass: rudderNode
-objectClass: rudderPolicyServer
-cn: ${RELAY_UUID}
-isBroken: FALSE
-isSystem: TRUE
-nodeId: ${RELAY_UUID}
-description: A relay server
-EOF
-${ECHO} " Done."
-
-#######################################################################################################################
-## Node groups (including special target) 
-#######################################################################################################################
-
-${ECHO} -n "INFO: Adding special group entries for relay server..."
-${LDAPADD} <<EOF >/dev/null 2>&1 || echo -n " entries already exist."
-dn: nodeGroupId=hasPolicyServer-${RELAY_UUID},groupCategoryId=SystemGroups,groupCategoryId=GroupRoot,ou=Rudder,cn=rudder-configuration
-objectClass: nodeGroup
-objectClass: top
-cn: All classic Nodes managed by ${RELAY_UUID} policy server
-description: All classic Nodes known by Rudder directly connected to the ${RELAY_UUID} server. This group exists only as internal purpose and should not be used to configure Nodes.
-isDynamic: TRUE
-nodeGroupId: hasPolicyServer-${RELAY_UUID}
-isSystem: TRUE
-isEnabled: TRUE
-jsonNodeGroupQuery: {"select":"nodeAndPolicyServer","composition":"And","where":[{"objectType":"node","attribute":"policyServerId","comparator":"eq","value":"${RELAY_UUID}"},{"objectType":"node","attribute":"agentName","comparator":"eq","value":"cfengine"}]}
-${EXISTING_NODEID}
-
-dn: ruleTarget=policyServer:${RELAY_UUID},groupCategoryId=SystemGroups,groupCategoryId=GroupRoot,ou=Rudder,cn=rudder-configuration
-objectClass: specialRuleTarget
-objectClass: top
-ruleTarget: policyServer:${RELAY_UUID}
-cn: ${RELAY_UUID} policy server
-description: Only the ${RELAY_UUID} policy server
-isEnabled: TRUE
-isSystem: TRUE
-
-EOF
-${ECHO} " Done."
+	${ECHO} "Done."
 
-#######################################################################################################################
-## Directives
-#######################################################################################################################
+	${ECHO} -n "INFO: Triggering promises generation..."
+	curl --proxy '' -k "https://localhost/rudder/api/deploy/reload"
```

Comment:
why not use DOWNLOAD_COMMAND here and why 127.0.0.1 at one place and localhost in the other one
I prefer localhost